getting-started/web-app.md

```diff
@@ -20,10 +20,30 @@ This article explains how to use the Telerik UI for Blazor components into a <a
 
 1. Select the [**Blazor Web App**](https://learn.microsoft.com/en-us/aspnet/core/blazor/project-structure?view=aspnetcore-8.0#blazor-web-app) project type, enter a name for your project, and then click **Next**.
 
-1. Select the .NET 8 framework, the desired [interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and interactivity location (per page or global).
+1. Select the desired [Interactive render mode](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes) and Interactivity location (*Global* or *Per page / component*). See the [notes below](#interactivity-considerations) while choosing Interactivity location.
 
 1. Click **Create**.
 
+### Interactivity Considerations
+
+Take into account the following notes when choosing the *Interactivity location* of your Telerik Blazor app:
+
+* We recommend *global* interactivity location for easier setup and usage of the Telerik components.
```

Comment:
Use bold for the interactivity locations so that the style makes it clear that you refer to the options in the UI. Keep the original style of the options (capitalization and spaces).